### PR TITLE
Add seiso standard tests

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
Add some initial Seiso standard tests

## Pull Request Checklist
Thank you for submitting a contribution to `goat`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
- [ ] Rebase your branch against the latest commit of the target branch, which likely should be `main`
- [ ] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

**Comment by JonZeolla** [comment]
Reminder that there are plans for a future feature that allows additional words to get loaded in at the repo level

File: etc/spelling.json [hunk 1]
```diff
@@ -1,20 +1,149 @@
...
-    "PCI-DSS",
-    "FedRAMP",
-    "SOW",
-    "MSA"
+    "aircrack",
+    "aireplay",
+    "airodump",
+    "axample",
+    "bau",
+    "bigip",
```
Comment (jimmio) [inline etc/spelling.json:31]:
```suggestion
    "bigip",
    "bssid",
    "bssids",
```

File: etc/spelling.json [hunk 2]
```diff
@@ -1,20 +1,149 @@
...
+    "bruteforcing",
+    "casp",
+    "cissp",
+    "connstring",
+    "cors",
+    "crackmapexec",
+    "csrf",
+    "cvss",
+    "dast",
+    "deauthenticate",
```
Comment (jimmio) [inline etc/spelling.json:44]:
```suggestion
    "deauthenticate",
    "deauthenticated",
    "deauthenticating",
```

File: etc/spelling.json [hunk 3]
```diff
@@ -1,20 +1,149 @@
...
+    "deauthentication",
+    "dehashed",
+    "dialoghandler",
+    "dmarc",
+    "dnsdumpster",
+    "domainkey",
+    "dpapi",
```
Comment (jimmio) [inline etc/spelling.json:53]:
```suggestion
    "dpapi",
    "eaphammer",
```

File: etc/spelling.json [hunk 4]
```diff
@@ -1,20 +1,149 @@
...
+    "fedramp",
+    "ffiec",
+    "gcux",
+    "gcwn",
+    "giac",
+    "gitleaks",
+    "gpen",
+    "gppa",
+    "gssp",
+    "hashcat",
```
Comment (jimmio) [inline etc/spelling.json:69]:
```suggestion
    "hashcat",
    "haveibeenpwned",
```

File: etc/spelling.json [hunk 5]
```diff
@@ -1,20 +1,149 @@
...
+    "infra",
+    "iso",
+    "johntheripper",
+    "kerberoast",
+    "kerberoasting",
+    "llmnr",
+    "lm",
+    "lsass",
+    "meraki",
+    "microsoft",
```
Comment (jimmio) [inline etc/spelling.json:91]:
```suggestion
    "microsoft",
    "mimikatz",
```

File: etc/spelling.json [hunk 6]
```diff
@@ -1,20 +1,149 @@
...
+    "mitm",
+    "msa",
+    "nbns",
+    "netbios",
+    "netlogon",
+    "netntlm",
+    "netntlmv",
+    "nmap",
+    "notpetya",
+    "ntlm",
```
Comment (jimmio) [inline etc/spelling.json:103]:
```suggestion
    "ntlm",
    "ntlmv",
```

File: etc/spelling.json [hunk 7]
```diff
@@ -1,20 +1,149 @@
...
+    "meraki",
+    "microsoft",
+    "minidump",
+    "mitm",
+    "msa",
+    "nbns",
+    "netbios",
+    "netlogon",
+    "netntlm",
+    "netntlmv",
```
Comment (jimmio) [inline etc/spelling.json:100]: Does this mean it will catch "netntlmv2"?

File: etc/spelling.json [hunk 8]
```diff
@@ -1,20 +1,149 @@
...
+    "servdev",
+    "sha256",
+    "sha256sum",
+    "smbclient",
+    "smbcreds",
+    "snaffler",
+    "sophos",
+    "sow",
+    "spns",
+    "sshkeys",
```
Comment (jimmio) [inline etc/spelling.json:136]:
```suggestion
    "sshkeys",
    "ssid",
    "ssids",
```

File: etc/spelling.json [hunk 9]
```diff
@@ -1,20 +1,149 @@
...
+    "telerik",
+    "thycotic",
+    "totp",
+    "trufflehog",
+    "typosquat",
+    "typosquatting",
+    "unmarshal",
+    "unmarshalling",
+    "vectr",
+    "vulnerabilites",
```
Comment (jimmio) [inline etc/spelling.json]:
```suggestion
    "vulnerabilities",
```

File: etc/spelling.json [hunk 10]
```diff
@@ -1,20 +1,149 @@
...
+    "wapt",
+    "whoami",
```
Comment (jimmio) [inline etc/spelling.json:151]:
```suggestion
    "whoami",
    "wireshark",
```

File: etc/spelling.json [hunk 11]
```diff
@@ -1,20 +1,149 @@
...
+    "wpad",
+    "xss"
```
Comment (jimmio) [inline etc/spelling.json]:
```suggestion
    "xss",
    "xxe"
```

File: etc/spelling.json [hunk 12]
```diff
@@ -1,20 +1,149 @@
...
+    "dnsdumpster",
+    "domainkey",
+    "dpapi",
+    "ept",
+    "essid",
+    "essids",
+    "fedramp",
+    "ffiec",
+    "gcux",
+    "gcwn",
```
Comment (jimmio) [inline etc/spelling.json:62]:
```suggestion
    "gcwn",
    "getuserspns",
```

**Review by jimmio — CHANGES_REQUESTED** [review]
Added some dictionary suggestions. Also having trouble running `goat` locally. When I run `invoke goat --local` inside the `pipenv` shell, I get `No idea what '--local' is!`.

File: etc/spelling.json (hunk 7 above)
Comment (JonZeolla) [reply etc/spelling.json:100]: Yes, I'm fairly certain the way `cspell` works would split NetNTLMv2 into "netntlmv" and "2", and then ignore "2" for being a number and < 4 characters.

**Comment by JonZeolla** [comment]
@jimmio you should be all set for a new look.  Run `pipenv run invoke goat` instead now; no longer need to differentiate local vs github actions run.

**Review by jimmio — APPROVED** [review]
LGTM, ran successfully and passed all tests. Initially it was timing out on the security tests, which I fixed by giving Docker more resources.

**Pull request merged**